src/xmipp/libraries/reconstruction/volume_subtraction.cpp

```diff
@@ -208,37 +217,36 @@ class ProgVolumeSubtraction: public XmippProgram
     	for (int n=0; n<iter; ++n)
     	{
     		std::cout<< "---Iter " << n << std::endl;
```

Comment:
what is happening in this for loop?

Reply:
This loop is to adjust the volumes (make them as similar as possible in an iterative way) before the subtraction. We can see it in detail if you prefer.